LICENSE

```diff
@@ -1,4 +1,5 @@
 Apache License
```

Comment:
I think this just needs to be centered, and brought down a line?  see: https://github.com/linuxkit/linuxkit/blob/master/LICENSE